### ISSUE TITLE
Improve UX/UI of Transaction list in VENFT App

### ISSUE THREAD

**Issue by fyziktom** [fyziktom/VirtualEconomyFramework#55]
The [transaction list](https://github.com/fyziktom/VirtualEconomyFramework/blob/main/VirtualEconomyFramework/VENFTApp-Blazor/Components/Account/TransactionsList.razor) needs redesign/refactoring (both html and C#).

![Actual Transaction list](https://ipfs.infura.io/ipfs/QmbeuNbtp7Atujbs6Sqpi2i6icv45ryZ7R1x9tyjGVhnWj)

This component has few related components/lists which makes sense together or usually are used together. 
It is [Transaction Details](https://github.com/fyziktom/VirtualEconomyFramework/blob/main/VirtualEconomyFramework/VENFTApp-Blazor/Components/Account/TransactionDetails.razor), which has connection in code now, and second are both [NFT Payment](https://github.com/fyziktom/VirtualEconomyFramework/blob/main/VirtualEconomyFramework/VENFTApp-Blazor/Components/NFTs/Payments/PaymentsList.razor) and [NFT Receipts](https://github.com/fyziktom/VirtualEconomyFramework/blob/main/VirtualEconomyFramework/VENFTApp-Blazor/Components/NFTs/Payments/ReceiptsList.razor) lists and details. These kinds of lists are related to more financial aspects of the App, thats why I think they should be considered together during design of new UX/UI for this part of the App (take this more like brainstorming please).

![NFT Payment and Receipts Lists](https://ipfs.infura.io/ipfs/QmX7L45uYdAWmEbA5515m5c6ySrcV3xctHVoj6sVwXotKC)

**Comment by Lowentwickler** [comment]
I suggest to create a generic component for tables, so we can ensure, that all tables in the applications works and looks the same.

Regarding the table itself, I suggest this list of columns:

1. Tx ID -> Link to explorer
2. Recepient -> Link to explorer
3. Sender -> Link to explorer
4. Type (Message, Receipt, Payment etc)
5. Date
6. Details button

Also as a part of this, duplicated entries should be solved, so only one is visible.

@fyziktom Do you agree?

**Comment by Xaniaro** [comment]
We can also create generic component for cards. Instead of:
```
<div class="card" style="margin-top:10px;">
    <div class="card-header" style="padding-top: 5px;padding-bottom: 5px;">
        <h5 class="text-center">Title Example</h5>
    </div>
    <div class="card-body" style="padding: 5px;padding-left: 20px;padding-right: 20px;padding-bottom: 10px;">
        Content Example
    </div>
</div>
```
We can write this:
```
<Card style="margin-top:10px;" Title="Title Example">
    Content Example
</Card>
```
Or if we want to have more control:
```
<Card style="margin-top:10px;">
    <Header>
        <h5 class="text-center">Title Example</h5>
    </Header>
    <Body>
        Content Example
    </Body>
</Card>
```

**Comment by Lowentwickler** [comment]
From my experience Card/Header/Body is a good practice.
I would also recommend to use [tailwindcss](https://tailwindcss.com/) for this project and forbid to use "style" tag.

**Comment by fyziktom** [comment]
> I suggest to create a generic component for tables, so we can ensure, that all tables in the applications works and looks the same.
> 
> Regarding the table itself, I suggest this list of columns:
> 
>     1. Tx ID -> Link to explorer
> 
>     2. Recepient -> Link to explorer
> 
>     3. Sender -> Link to explorer
> 
>     4. Type (Message, Receipt, Payment etc)
> 
>     5. Date
> 
>     6. Details button
> 
> 
> Also as a part of this, duplicated entries should be solved, so only one is visible.
> 
> @fyziktom Do you agree?

It makes sense for me. good point :)

**Comment by fyziktom** [comment]
> From my experience Card/Header/Body is a good practice. I would also recommend to use [tailwindcss](https://tailwindcss.com/) for this project and forbid to use "style" tag.

I dont have lots of experiences with UI, so if you think it can reduce the style tags, it would be nice.
Main point of the VEFramework is to keep it simple without lots of dependencies which we cannot manage/maintain by ourselfs in the future.

**Comment by fyziktom** [comment]
Solved in #101. Now available as VEBlazor library.
